app/models/step.rb

```diff
@@ -10,6 +10,7 @@ class Step < ActiveRecord::Base
 
   belongs_to :user
   belongs_to :proposal
+  belongs_to :completed_by, class_name: "User"
```

Comment:
I'm not attached to the naming convention I used. I just wanted to have a method where the user who completed the step could be accessed via a standard method name that was backwards compatible. So I started with the `completer` method and worked backwards in terms of naming.